app/src/main/AndroidManifest.xml

```diff
@@ -74,15 +73,46 @@
 
         <activity
             android:name=".activity.BrowserActivity"
-            android:theme="@style/FullScreen">
+            android:theme="@style/FullScreen"
+            android:label="">
             <intent-filter>
                 <action android:name="org.openobservatory.ooniprobe.OPEN_BROWSER" />
-
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
-        <activity android:name=".activity.NotificationDialog"
+        <activity
+            android:name=".activity.NotificationDialog"
             android:theme="@style/Theme.Transparent" />
+        <activity
+            android:name=".activity.RunTestActivity"
+            android:label="@string/app_name"
+            android:launchMode="singleInstance">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <action android:name="org.openobservatory.ooniprobe.nettest" />
+                <data android:scheme="https"
+                    android:host="run.ooni.io"
+                    android:pathPrefix="/nettest" />
+            </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <action android:name="org.openobservatory.ooniprobe.nettest" />
+                <data android:scheme="http"
```

Comment:
Do we want to support `http://run.ooni.io` links? Cc: @hellais

Reply:
Ping @hellais: do we want to support non encrypted `run.ooni.io` links? I guess the possible reason why we want to do that is that one _may_ write the link without `https`. Do we really want this or do we want to enforce `https`?

Reply:
Yes I think we also want to support `http` links. I don't think it's a security concern as actually if the app is installed it will never hit the network. We do generate them by default as `https`, but I think it's best in the app to support both to avoid the case where users mistakenly use a http link and we will only trigger the app upon redirect (so if `ooni.io` is blocked nothing will be triggered)